dolphin/task_manager/manager.py

```diff
@@ -25,6 +25,7 @@
 from dolphin import manager
 from dolphin.task_manager import rpcapi as task_rpcapi
 from dolphin import coordination
+from dolphin.exporter import example_exporter
```

Comment:
Third_party exporter(example_exporter here ) can't be imported by task_manager, because task_manager should not be changed for each third_party exporter support.
task_manager should only use the exporter interface to connect and send data , there can be different implementation of exporters.

Reply:
In this example, task_manager only depends on report_example_data which is static, that means it will not change when a new third_party exporter  add. This is just an example, you may need to add report_alert_data, report_performance_data for real use cases, and you can seperate the definition of reprot_xxx_data and xxxExporter into different files.

Reply:
I just reorganized the example, it may be more clearer for understanding.

Reply:
Ok, thanks, Gives more clarity..